api/profile/profileRouter.js

```diff
@@ -4,6 +4,8 @@ const Parents = require('../parent/parentModel');
 
 const { ops } = require('../../lib');
 
+
```

Comment:
?

Reply:
Are 2 blank lines the only diff?

Reply:
Not entirely sure how that happened, but I did fix